### DISCUSSION TITLE
Battery Save command

### DISCUSSION THREAD

**Discussion by double1968** [cmcgerty/Sofar2mqtt#12] [General]
I am having this problem
I use my HYD-3600-ES inverter with fw 2.70 in passive mode, when I send the mqtt 'AUTO' command in 'BATTERY SAVE' mode, the inverter charges the battery if possible and does not discharge it only for a certain time that I have not yet understood and then goes into 'AUTO' mode with battery discharge and charge without having given any other commands.
Do others have this situation?

**Comment by cmcgerty** [comment]
As a safety measure, Sofar2mqtt puts the inverter in standard Auto mode during start up. So it’s possible your ESP8266 might be restarting after a while and hence sending an Auto commend.

The way that I use it is that once per minute, a routine in Node-Red determines what the current mode should be, so the situation you describe would not happen in my system so I can’t be much help, but consider using a repeating routine to send a regular instruction rather than only when you want to switch modes.

**Comment by double1968** [comment]
OK thank you .
I try to implement this function.

**Comment by cmcgerty** [comment]
@double1968 

Apologies to not picking up this as a bug when you first reported it. Battery_save is fixed in version 2.0.1 and later.

**Reply by double1968** [reply]
@cmcgerty 

Thanks, I will try to update
